src/ts/components/settings/DynamicSettingsPanelItem.ts

```diff
@@ -102,7 +102,8 @@ export class DynamicSettingsPanelItem extends SettingsPanelItem<DynamicSettingsP
     const handleItemClick = () => {
       this.displayItemsSubPage();
     };
-    this.getDomElement().on('click', (e) => {
+    this.getDomElement().on('click touchend', (e) => {
```

Comment:
@dweinber found an issue with this approach. The touchend triggers and opens the sub menu when releasing the touch after scrolling within the settings panel.